packages/components/inputs/async-select-input/src/async-select-input.tsx

```diff
@@ -347,8 +364,20 @@ const AsyncSelectInput = (props: TAsyncSelectInputProps) => {
     componentName: 'AsyncSelectInput',
   });
 
-  const placeholder =
-    props.placeholder || intl.formatMessage(messages.placeholder);
+  // const placeholder =
+  //   props.placeholder || intl.formatMessage(messages.placeholder);
+
+  const placeholder = props.placeholder
+    ? props.placeholder
+    : props.appearance === 'filter'
+    ? 'Search'
+    : intl.formatMessage(messages.placeholder);
+
+  // TODO: uncomment to replace placeholder logic once select-input PR for this has been merged.
```

Comment:
When the select-input PR is merged, I will uncomment this since they will be using the same messages keys.